common/ActionManager.js

```diff
@@ -25,12 +47,11 @@ class ActionManager {
   defineActions(definitions) {
     Object.keys(definitions).forEach(name => {
       const definition = definitions[name];
-      const composed = function() {
-        return compose([
+      const composed = (...args) =>
```

Comment:
Sorry, I meant spread syntax in function calls 😜  I could have sworn this was busted..... but maybe I was remembering wrong?